astropy/analytic_functions/tests/test_blackbody.py

```diff
@@ -32,7 +32,7 @@ def test_blackbody_scipy():
 
     """
     flux_unit = u.Watt / (u.m ** 2 * u.um)
-    wave = np.logspace(0, 8, 1e6) * u.AA
+    wave = np.logspace(0, 8, 100000) * u.AA
```

Comment:
Thanks, @mhvk ! It's a holiday weekend in USA, so I wouldn't have gotten to this till Tuesday anyway. I have the same question as @astrofrog above.